pkg/jsonstream/image_push.go

```diff
@@ -0,0 +1,98 @@
+package jsonstream
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/containerd/containerd/remotes/docker"
+)
+
+// Pushjobs defines a job in upload progress
+type Pushjobs struct {
```

Comment:
use the camel way of `PushJobs` instead of `Pushjobs`?

Reply:
Updated